File: LinearSearch/LinearSearch.go
package main

import "fmt"

func main(){
	fmt.Println("Linear Search in Go")
	myArr := []int{1,2,3,4,5}
	index := linearSearch(myArr, 4)
	fmt.Println(index)
}

func linearSearch(arr []int, target int) int {

	for i := range len(arr) {
		if arr[i] == target {
			return i
		}
	}

	return -1
}

File: Algos/LinearSearch.java
package Algos;

public class LinearSearch {
    public static void main(String[] args) {
        int arr[] = {1,2,3,4,5,6,7};
        int index = findNum(arr, 5);
        System.out.println(index);
    }
    static int findNum(int[] arr, int target){
        int ans = -1;
        for(int i = 0; i < arr.length; i++){
            if(arr[i] == target){
                ans = i;
            }
        }
        return ans;
    }
}
